OpenRA.Game/Game.cs

```diff
@@ -870,9 +871,9 @@ public static int CreateLocalServer(string map)
 				AllowPortForward = false
 			};
 
-			server = new Server.Server(new IPEndPoint(IPAddress.Loopback, 0), settings, ModData, false);
+			server = new Server.Server(new IPEndPoint(Socket.OSSupportsIPv6 ? IPAddress.IPv6Loopback : IPAddress.Loopback, 0), settings, ModData, false);
```

Comment:
While a reverse dns lookup for `localhost` would be the most future proof method, this would potentially be more stable in case of a misconfigured operating system installation. I am not completely sure which one we want to prefer.

Reply:
Be careful with introducing DNS lookups.  We have had several issues over the years with this stalling the for potentially many hundreds (or more!) of milliseconds.

Reply:
In this case it shouldn't matter because `localhost` should be in the hosts file. I don't know about Windows but that's how it is on Linux. Even if it isn't, it is a one time stall which shouldn't be that bad.

Reply:
The Window$ 7 `hosts` file says **`localhost` resolution is handled within DNS itself**, so there shouldn't be a worry about that. You'll need a working networking for ORA multiplayer anyway ...

Reply:
> You'll need a working networking for ORA multiplayer anyway ...

This is the part of the code that hosts missions and skirmish games :wink: